src/chitterApi.js

```diff
@@ -0,0 +1,103 @@
+class chitterApi {
+  constructor() {
+    this.apiUrl = "https://chitter-backend-api-v2.herokuapp.com/peeps";
+    this.apiCreateUserUrl =
+      "https://chitter-backend-api-v2.herokuapp.com/users";
+    this.apiLoginUserUrl =
+      "https://chitter-backend-api-v2.herokuapp.com/sessions";
+    this.apiCreatePeppUrl =
+      "https://chitter-backend-api-v2.herokuapp.com/peeps";
+    this.apiLikePeepUrl =
+      "https://chitter-backend-api-v2.herokuapp.com/peeps/peepId/likes/userId";
+    this.apiDeletePeepUrl =
+      "https://chitter-backend-api-v2.herokuapp.com/peeps/peepId";
+  }
+  async fetchAll() {
+    return await fetch(this.apiUrl)
+      .then((response) => response.json())
+      .then((response) => {
+        return response;
+      });
```

Comment:
Check `.catch` - it's good practice to log errors.

Reply:
Try catch yes

Reply:
not try catch, but a `.catch` that you can use on promises - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/catch